public/assets/styles/_sections/RequestBeta.less

```diff
@@ -0,0 +1,49 @@
+#RequestBeta {
+  padding: 2em 0;
+  p {
+    font-size: 28px;
+    letter-spacing: -0.12px;
+    line-height: 42px;
+  }
+  .inputContainer {
+    width: 100%;
+    .col-xs-12 {
```

Comment:
It's typically a good idea to not use a utility class name that exists for 1 certain purpose directly in your own stylesheets. The reason being, you may want to change `.col-xs-12` in the markup to `.col-md-12` or something related, and then your own styles here would break. To remedy that, I'd add another classname to this element, and use that custom classname here instead of `.col-xs-12`.